packaging/observability/opentelemetry-collector-medium.yaml

```
@@ -0,0 +1,70 @@
+# This configuration exports:
+# - Container, Pod, Volume, and Node metrics
+# - Kubernetes Events (Warnings only)
+# - MicroShift service journald
+
+receivers:
+  kubeletstats:
+    auth_type: tls
+    ca_file: /var/lib/microshift/certs/ca-bundle/client-ca.crt
+    key_file: /var/lib/microshift/certs/admin-kubeconfig-signer/openshift-observability-client/client.key
+    cert_file: /var/lib/microshift/certs/admin-kubeconfig-signer/openshift-observability-client/client.crt
+    insecure_skip_verify: true
+    collection_interval: 10s
+    endpoint: "${env:K8S_NODE_NAME}:10250"
+    node: ${env:K8S_NODE_NAME}
+    k8s_api_config:
+      auth_type: kubeConfig
+  k8s_events:
+    auth_type: kubeConfig
+  journald:
+    units:
+      - microshift
+processors:
```

So it's like the `small.yaml` - the `receivers` and `processors` are distinct sections separated by newline for those who read too quickly and skip lines :D

```suggestion

processors:
```